README.md

```diff
@@ -322,14 +322,15 @@ module.exports = {
 }
 
 // test.js
-var stubs = {
-  './baz': {
-    method: function(val) {
-      console.info('goodbye');
-    },
-    '@global': true
+var bazStub = {
+  method: function() {
+    console.info('goodbye');
   }
 };
+  
+var stubs = {
+  './baz': Object.assign(bazStub, {'@global': true}) 
```

Comment:
I don't think it improves the docs to make this change—if anything it obfuscates the purpose of using `Object.assign`

Reply:
My main intention was to make it more explicit, which parts belong to the stub and what's required by proxyquire to stub this dependency globally. That's why I extracted the bazStub into a separate variable.
Given that, there was a need to add the `@global:true` - and I think using `Object.assign` for that is a pretty nice&consice way of doing that. However this is more a personal perference - so if you feel like it makes the docs harder to understand I'm happy to revert this change.

Reply:
I get it, I'd just rather leave both the simple single object literal approach in there in addition to presenting an example more suited to `Object.assign` further down. Your approach makes sense in practice but I don't mind retaining a more verbose example that demonstrates usage more so than intent.